apps/acme-cart/README.md

```diff
@@ -39,14 +39,20 @@ The **cart** service, either running inside a Docker container or as a stand-alo
 * **REDIS_PASSWORD**: The password to connect to Redis (defaults to `blank`)
 * **CART_PORT**: The port number the cart service will listen to requests (defaults to `5000`)
 
-The Docker image is based on the Bitnami Python container. Use this commands to run the latest stable version of the payment service with all available parameters:
+The Docker image of acme-cart can be build using the Dockerfile and following the command
 
 ```bash
-# Run the Redis container
-docker run -p 6379:6379 bitnami/redis:latest
+# Build the acme-cart image
+docker build -t acme-cart .
+```
+
+The Docker image is based on the Bitnami Python container.
```

Comment:
I think the image is the official python image and NOT the Bitnami.

Reply:
done